int32 round_time_remaining
int32 ct_score
int32 t_score
string game_phase
bool bomb_planted
geometry_msgs/Point bomb_location
string[] dead_players
